meson.build

```diff
@@ -69,6 +71,15 @@ if enable_breadcrumbs
   add_project_arguments('-DVKD3D_ENABLE_BREADCRUMBS', language : 'c')
 endif
 
+if enable_tracy
+  add_project_arguments('-DVKD3D_ENABLE_PROFILING', language : 'c')
```

Comment:
This will probably conflict if enable_profiling is also set. enable_tracy should set that variable.

Reply:
Fixed